ee/cis/win-10/cis-policy-queries.yml

```diff
@@ -3233,6 +3233,187 @@ spec:
+  contributors: sharon-fdm
+---
+apiVersion: v1
+kind: policy
+spec:
+  name: CIS - Ensure 'Audit Security State Change' is set to include 'Success'
+  platforms: win10
+  platform: windows
+  description: |
+    This subcategory reports changes in security state of the system, such as when the security subsystem starts and stops. Events for this subcategory include:
+    -  4608: Windows is starting up.
+    -  4609: Windows is shutting down.
+    -  4616: The system time was changed.
+    -  4621: Administrator recovered system from CrashOnAuditFail. Users who are not
+    administrators will now be allowed to log on. Some audit-able activity might not have been recorded.
+    The recommended state for this setting is to include: 'Success'.
+  resolution: |
+    Automatic method:
+    Ask your system administrator to establish the recommended configuration via GP, set the following UI path to include Success:
+    'Computer Configuration\Policies\Windows Settings\Security Settings\Advanced Audit Policy Configuration\Audit Policies\System\Audit Security State Change'
+  query: |
+    SELECT 1 FROM mdm_bridge where mdm_command_input = "<SyncBody><Get><CmdID>1</CmdID><Item><Target><LocURI>./Device/Vendor/MSFT/Policy/Result/Audit/System_AuditSecurityStateChange</LocURI></Target></Item></Get></SyncBody>" 
+    AND (mdm_command_output = 3 OR mdm_command_output = 1);
+  purpose: Informational
+  tags: compliance, CIS, CIS_Level1, CIS_win10_enterprise_1.12.0, CIS_bullet_17.9.3
+  contributors: sharon-fdm
+---
+apiVersion: v1
+kind: policy
+spec:
+  name: CIS - Ensure 'Audit Security System Extension' is set to include 'Success'
+  platforms: win10
+  platform: windows
+  description: |
+    This subcategory reports the loading of extension code such as authentication packages by the security subsystem. Events for this subcategory include:
+    -  4610: An authentication package has been loaded by the Local Security Authority.
+    -  4611: A trusted logon process has been registered with the Local Security Authority.
+    -  4614: A notification package has been loaded by the Security Account Manager.
+    -  4622: A security package has been loaded by the Local Security Authority.
+    -  4697: A service was installed in the system.
+    The recommended state for this setting is to include: Success.
+  resolution: |
+    Automatic method:
+    Ask your system administrator to establish the recommended configuration via GP, set the following UI path to include Success:
+    'Computer Configuration\Policies\Windows Settings\Security Settings\Advanced Audit Policy Configuration\Audit Policies\System\Audit Security System Extension'
+  query: |
+    SELECT 1 FROM mdm_bridge where mdm_command_input = "<SyncBody><Get><CmdID>1</CmdID><Item><Target><LocURI>./Device/Vendor/MSFT/Policy/Result/Audit/System_AuditSecuritySystemExtension</LocURI></Target></Item></Get></SyncBody>" 
+    AND (mdm_command_output = 3 OR mdm_command_output = 1);
```

Comment:
same question as a before - perhaps we should just include "mdm_command_output = 1"

Reply:
Same answer.

Reply:
Same 🙂